addons/im_livechat/models/chatbot_script_step.py

```diff
@@ -431,10 +453,17 @@ def _process_step_forward_operator(self, discuss_channel, users=None):
 
         return posted_message
 
+    def _field_store_repr(self, field_name):
+        if field_name == "message":
+            return [
+                Store.Attr("message", lambda step: step.html_message),
+            ]
+        return [field_name]
+
     def _to_store_defaults(self):
         return [
             Store.Many("answer_ids"),
             Store.Attr("is_last", lambda step: step._is_last_step()),
-            Store.Attr("message", lambda s: plaintext2html(s.message) if s.message else False),
+            "message",
```

Comment:
The goal in that case would be to have `html_message` directly here, rather than using `_field_store_repr`.